pkg/webhooks/ingresscontroller/ingresscontroller.go

```diff
@@ -149,12 +176,151 @@ func (wh *IngressControllerWebhook) authorized(request admissionctl.Request) adm
 		}
 	}
 
-	ret = admissionctl.Allowed("IngressController operation is allowed")
+	/* TODO:
+	 * 1) ONLY check for privatelink clusters? How?
```

Comment:
I don't _believe_ this only needs to apply to PrivateLink clusters, I think it could apply to _any_ cluster

Reply:
Correct, it needs to check against any clusters.

Reply:
Thanks. Removed/updated comment.